lib/api/src/grpc/validate.rs

```diff
@@ -209,12 +209,34 @@ pub fn validate_collection_name(value: &str) -> Result<(), ValidationError> {
     }
 }
 
+/// Validate the first and the last coordinate of a Polygon is the same to form a closed polygon
+pub fn validate_geo_polygon(points: &Vec<GeoPoint>) -> Result<(), ValidationError> {
+    if points.len() < 4 {
+        let mut err = ValidationError::new("length");
+        err.add_param(Cow::from("min"), &4);
+        return Err(err);
+    }
+
+    let first_point = &points[0];
+    let last_point = &points[points.len() - 1];
+    if first_point != last_point {
+        let mut err = ValidationError::new("closed_polygon");
+        err.message.replace(Cow::from(
+            "The first and last points should be the same to form a closed polygon",
+        ));
```

Comment:
I think that, for consistency, it would be better to remove the message here and add it to the rest:

https://github.com/zzzz-vincent/qdrant/blob/77347abc046c3ccdf235245c9d48065481862ffb/lib/collection/src/operations/validation.rs#L51-L95

Reply:
done. Thanks!